sql/rowexec/show_iters.go

```diff
@@ -395,7 +395,6 @@ func (i *showCreateTablesIter) produceCreateTableStatement(ctx *sql.Context, tab
 	tableCollation := table.Collation()
 	for i, col := range schema {
 		var colDefaultStr string
-		// TODO: The columns that are rendered in defaults should be backticked
```

Comment:
Was this comment removed because it was obsolete?

Reply:
Yeah, we added a rule called `backtickDefaultColumnValueNames` a while ago.